src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2020,15 +2020,14 @@ bool processOnChangeConditionForSubscription
 *
 */
 BSONArray processConditionVector
-(
-  NotifyConditionVector*           ncvP,
+(NotifyConditionVector*           ncvP,
   EntityIdVector                   enV,
   AttributeList                    attrL,
-  std::string                      subId,
```

Comment:
Of course... but that is part of a next PR. The "strategy" is going PR by PR with small steps (due to each change involves many files).

NTC